### PR TITLE
Add cdnistt IANA registry

### PR DESCRIPTION
**Pull request by PSUdaemon**

### CODE REVIEW

File: draft-ietf-cdni-uri-signing.xml [hunk 1]
```diff
@@ -1262,6 +1263,37 @@ http://*/folder/content-83112371/quality_*/segment????.mp4
...
       </section>
+
+      <section anchor="sec.IANA.cdnistt"
+               title="CDNI URI Signing Signed Token Transport">
+        <t>The IANA is requested to create a new "CDNI URI Signing
+        Signed Token Transport" subregistry in the "Content
+        Delivery Networks Interconnection (CDNI) Parameters" registry.
+        The "CDNI URI Signing Signed Token Transport"
+        namespace defines the valid values
+        that may be included in a URI Signing token.  Additions to the
```
Comment (edrthomas) [inline draft-ietf-cdni-uri-signing.xml]: I am not sure that "URI Signing token" is the defined term in the specification. Maybe use "signed JWTs" or "Signed Token"?

Comment (PSUdaemon) [inline draft-ietf-cdni-uri-signing.xml]: That's meant to be two parts. The `CDNI URI Signing` `Signed Token Transport`. The other registries follow the same naming pattern. It doesn't flow, but I think it's accurate. I'm open to other word choice if you have a specific preference.

Comment (edrthomas) [inline draft-ietf-cdni-uri-signing.xml]: Sorry I wasn't clear enough.  The title of the registry is great. I just pointed out that the term "URI Signing token" at the end of the sentence is not a defined term in the specification. Although I do understand what it is meant, maybe not a normarl reader. We could change "URI Signing token" to "Signed Token" which is used throughout the specification. What do you think ?

Comment (PSUdaemon) [inline draft-ietf-cdni-uri-signing.xml]: Ok, I think that whole line is just wrong. I have pushed a new commit to fix.

**Pull request merged**